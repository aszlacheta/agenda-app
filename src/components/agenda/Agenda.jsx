import React from 'react';
import './Agenda.scss';

export default function Agenda() {

    return <div className="agenda">Agenda</div>
}