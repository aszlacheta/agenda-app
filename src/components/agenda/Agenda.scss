.agenda {
    display: flex;
    position: relative;
    flex-direction: column;
    justify-content: center;
    flex: 1;
    margin-top: 50px;
    margin-left: 150px;

    &:before {
        --width: 100%;
        content: "";
        position: absolute;
        top: -50px;
        left: calc(var(--width) * -1 + 30px);
        width: var(--width);
        height: 100%;
        border: 8px solid black;
        z-index: 0;
    }

    .agenda-container {
        overflow: auto;
    }
}

@media only screen and (max-width: 700px), (max-height: 500px) {
    .agenda {
        margin-left: 0px;
        margin-top: 20px;

        &::before {
            top: -20px;
        }
    }
}
