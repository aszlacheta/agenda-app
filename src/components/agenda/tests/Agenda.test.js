import React from 'react';
import renderer from 'react-test-renderer';
import Agenda from '../../agenda/Agenda';

describe('Agenda', () => {
    it('check if renders properly', () => {
        const tree = renderer
            .create(<Agenda />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})