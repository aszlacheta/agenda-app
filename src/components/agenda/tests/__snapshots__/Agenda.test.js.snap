// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agenda check if renders properly 1`] = `
<div
  className="agenda"
>
  Agenda
</div>
`;
